docs/dev/tests/paypal.rst

```diff
@@ -0,0 +1,38 @@
+Paypal testing
+==============
+
+To test paypal payments you need to:
+
+* Create developer account
```

Comment:
You can use 

    .. contents::
       :local:

to generate menu automatically